src/utils/PushNotification-parse.ts

```diff
@@ -238,7 +238,8 @@ export const parse = async (
     console.log(
       'SIP PN debug: PushNotification-parse: local notification missed call',
     )
-    if (as.phoneappliEnabled()) {
+
+    if (as.userExtensionProperties && as.phoneappliEnabled()) {
```

Comment:
Update check userExtensionProperties to handle action move to phone Appli app or recents tab